docs/faq/repositories/i-renamed-my-repository-on-the-git-provider.md

```diff
@@ -2,6 +2,6 @@
 
 If you changed the name or URL of your repository on your Git provider, you can update the name and URL of the repository on Codacy to point to the new location. This ensures that you won't lose historical data about your repository on Codacy.
 
-To rename your repository on Codacy, open the page **Settings** and click the button **Update name**.
+To rename your repository on Codacy, open the page **Settings** and click the button **Update repository**.
```

Comment:
Here the label of the button changes, as well as its capabilities. We used to have two buttons here.

I'm opting not to expand on this, given that the context of the page.